spec/pluck_to_tstruct_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'rails_helper'
 require 'sorbet-rails/model_rbi_formatter'
+require "tstruct_comparable"
```

Comment:
```suggestion
require 'tstruct_comparable'
```

for consistency with the rest of the file